tests/actions/test_urls.py

```diff
@@ -0,0 +1,26 @@
+def test_debug_toolbar_urls_enabled_when_corresponding_settings_set_to_true(
+    settings, remove_from_module_cache
+):
+    with remove_from_module_cache("actions.urls"):
+        settings.DEBUG = True
+        settings.DEBUG_TOOLBAR = True
+
+        # Only import actions.urls after settings are patched
+        import actions.urls
+
+        pattern = str(actions.urls.urlpatterns[-1].pattern)
```

Comment:
Must DDT's URL pattern be the last? If so, then an explanation would be useful. If not, then I think we could parameterise this test and merge it with the next.